master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/index.md

```diff
@@ -144,7 +187,7 @@ Below are a few examples for how to get started.
 
 ## Configuration details
 
-The following environment variable configuration options are supported by the various {{site.prodname}} components **when running without etcd**.
+The following environment variable configuration options are supported by the various {{site.prodname}} components when running with Kubernetes as the datastore.
```

Comment:
We have reached an agreement to call this "Kubernetes API datastore" so it might be nice if time permits to try to standardize on that, if possible